pvlib/albedo.py

```diff
@@ -0,0 +1,103 @@
+"""
+The ``albedo`` module contains functions for modeling albedo.
+"""
+
+from pvlib.tools import sind
+
+WATER_COLOR_COEFFS = {
+    'clear_water_no_waves': 0.13,
+    'clear_water_ripples_up_to_2.5cm': 0.16,
+    'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps': 0.23,
+    'clear_water_frequent_whitecaps': 0.3,
+    'green_water_ripples_up_to_2.5cm': 0.22,
+    'muddy_water_no_waves': 0.19
+}
+
+WATER_ROUGHNESS_COEFFS = {
+    'clear_water_no_waves': 0.29,
+    'clear_water_ripples_up_to_2.5cm': 0.7,
+    'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps': 1.23,
+    'clear_water_frequent_whitecaps': 2,
+    'green_water_ripples_up_to_2.5cm': 0.7,
+    'muddy_water_no_waves': 0.29
+}
+
+
+def albedo_water(solar_elevation, color_coeff=None, wave_roughness_coeff=None,
+                 surface_condition=None):
+    r"""
+    Estimation of albedo for inland water bodies.
+
+    The available surface conditions are for inland water bodies, e.g., lakes
+    and ponds. For ocean/open sea, an albedo value of 0.06 is recommended.
+    See :const:`pvlib.irradiance.SURFACE_ALBEDOS`.
+
+    Parameters
+    ----------
+    solar_elevation : numeric
+        Sun elevation angle. [degrees]
+
+    color_coeff : float, optional
+        Water color coefficient. [-]
+
+    wave_roughness_coeff : float, optional
+        Water wave roughness coefficient. [-]
+
+    surface_condition : string, optional
+        If supplied, overrides ``color_coeff`` and ``wave_roughness_coeff``.
+        ``surface_condition`` can be one of the following:
+        'clear_water_no_waves', 'clear_water_ripples_up_to_2.5cm',
+        'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps',
+        'clear_water_frequent_whitecaps', 'green_water_ripples_up_to_2.5cm',
+        'muddy_water_no_waves'.
```

Comment:
```suggestion
        ``'clear_water_no_waves'``, ``'clear_water_ripples_up_to_2.5cm'``,
        ``'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps'``,
        ``'clear_water_frequent_whitecaps'``, ``'green_water_ripples_up_to_2.5cm'``,
        ``'muddy_water_no_waves'``.
```

Maybe in a list it could be read easier 🤔

Reply:
a bulleted list like this may be a good option: https://github.com/pvlib/pvlib-python/blob/main/pvlib/iam.py#L667-L675

Reply:
I like the bulleted list. Remember that you need the vertical blank space for the list to render correctly.